scripts/aggregate-changelogs/script.py

```diff
@@ -300,8 +305,9 @@ def generate_release_file(repo_shortname, repo_config, release):
         catfile.write(dump(categories, Dumper=CDumper))
 
     # Write changelog items
-    for repo_short in conf['repositories']:
-        repo_conf = conf['repositories'][repo_short]
+    for repo_short in sorted(conf['repositories']):
+        print(f'Repo {repo_short}')
```

Comment:
do you want to keep this one?

Reply:
Yes. It helps with manual testing to see which repo is processed. That's also why I changed to `sorted` for iteration in order to achieve a stable order.